### DISCUSSION TITLE
Create Field To Specify SSH Profile During Cloning Of Repository

### DISCUSSION THREAD

**Discussion by ThePieMonster** [gitextensions/gitextensions#11095] [Q&A]
### Feature description

I recently came to find that there is no way to use the GUI in Git Extensions when wanting to setup / clone a repository with the use of SSH.

`SSH Profile - GitHub Repository - Windows Path`

`git clone git@github-profile:User/Repo.git File\Path\On\PC\`

I think the best place for this addition would be somewhere on the "Clone repo" screen and perhaps putting a section where the dark red bar is that I drew below would be a good spot. There could be a text bar here that would allow the user to enter in the SSH profile path or a dropdown that auto populates the profiles found in ~/.ssh/<profiles>.

![image](https://github.com/gitextensions/gitextensions/assets/11359134/fd698cf3-a76b-4b8a-a0f1-e3c66e5cd896)

### Environment

- GitExtensions version: 4.1.0
- GIT version: 2.41.0
- OS version: Windows 11

**Comment by vbjay** [comment]
Change ssh settings for putty and you will see a place to put a putty key file.

**Reply by RussKie** [reply]
![image](https://github.com/gitextensions/gitextensions/assets/4403806/eb6ef0c1-eafe-45b8-8f68-5f94c264835f)

![image](https://github.com/gitextensions/gitextensions/assets/4403806/5d5ef91b-32d6-457f-8ca1-2e2d64e64e8a)
